### DISCUSSION TITLE
Main Domain Website Not Showing

### DISCUSSION THREAD

**Discussion by vreykin** [vercel/community#2115] [Help] [answered]
### Summary

I configured my domain to Cloudflare to protect my website and configure my DNS more quickly. Still, I also added Vercel and connected to my 3rd party Cloudflare account, the Vercel project is connected to my Github, which configures all my sites, the default domain for Vercel, shows my website, on the other hand, the main domain that I bought, It doesn't show up, I waited for about 4 days and it still doesn't work, could anybody please assist me?

Edit: Yes, I did configure Cloudflare to Vercel.

**Comment by amyegan** [comment, marked as answer]
Hi @Vitlyr! I don't want to make any assumptions. Did you follow the steps from our documentation to [add and configure a custom domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#)? If so, the domain should have a "Valid Configuration" indicator on the Domains page in your Project Settings.

<img src="https://vercel.com/_next/image?url=%2Fstatic%2Fdocs%2Fconcepts%2Fprojects%2Fcustom-domains%2Fconfigured-domain-dark.png&w=750&q=75" alt="custom domain valid configuration" />

If you see "Invalid Configuration", then DNS record propagation may be delayed or something might need to be adjusted in your Cloudflare records. You can find more info related to some common scenarios in our guide on [how to use a Cloudflare domain with Vercel](https://vercel.com/guides/using-cloudflare-with-vercel)
